src/Component/subBreedList.tsx

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { FunctionComponent, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import LoadImg from './LoadImg'
+
+interface SubBreedListProps {
+    main:string
+}
+ 
+const SubBreedList: FunctionComponent<SubBreedListProps> = (props:SubBreedListProps) => {
+    let [subBreeds,setSubBreeds] = useState<string[]>([])
+    useEffect( () => {
+        axios
+        .get('https://dog.ceo/api/breed/'+props.main+'/list')
+        .then(res=>{
+            setSubBreeds(res.data.message);
+        })
+        .catch(err => {console.log(err);})

```

Comment:
Could you abstract this api call outside the component?

Reply:
Hey sorry for the late reply.

The idea of using redux crossed my mind but I decided to use Hooks because I saw this project as a learning opportunity.

Yes, I should and could abstract the API call and I was planning to do it but I didn't do it because the CSS layout troubled me a little more than what I anticipated. If you want me I can def address this issue for every component.

Reply:
The structuring is awful. Now that I decided to refactor my code I see a lot of hardcoding and useless files.

The idea I initially had was to split the filesystem to 
a)Pages Components
b)Specific Components attached to a view 
c) General Components that will be used as modules to a plethora of views.